website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,96 @@
+## Usage
+
+### When to use
+
+- When displaying code examples and longer snippets of code that benefit from syntax highlighting.
+
+### When not to use
+
+- As a full-blown code editor.
+
+### When to use a CodeBlock vs a CopySnippet
+
+There is some overlap in the copying functionality of the CopySnippet and the CodeBlock. Which to use generally comes down to the complexity of the code/value displayed within the component, and whether the user benefits from seeing the larger context of the code example.
+
+Use a CodeBlock if:
```

Comment:
Should the "if" be removed here, since the following paragraphs also start with an "if" (or viceversa) 
```suggestion
Use a CodeBlock:
```

Same for the block below.

Reply:
Yep, that's a good callout, thank you!